rules/windows/defense_evasion_code_injection_conhost.toml

```diff
@@ -33,11 +31,12 @@ event.category:process and event.type:(start or process_started) and
 framework = "MITRE ATT&CK"
 [[rule.threat.technique]]
 id = "T1093"
-name = "Process Injection"
+name = "Process Hollowing"
```

Comment:
I think we need to update this rule. looks like it should be injection not hollowing.

Reply:
https://attack.mitre.org/techniques/T1055/